portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,28 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const fact1 = 'I talk in my sleep'
+  const fact2 = 'I have a dog named Yuki'
+  const fact3 = 'I used to be afraid of ceiling light reflections on the floor'
+  const fact4 = 'I live to eat'
+  const fact5 = 'Japan is my favourite holiday destination'
+
+  const facts = [fact1, fact2, fact3, fact4, fact5];
```

Comment:
since fact1 - 5 is only used in facts. I think it is better to remove the variable declaration for fact 1-5
e.g
const facts  = [
     'I talk in my sleep',
     ....
]

Reply:
Was thinking more along the lines of declaring these facts as constants but I see how this would be better. Have changed the code to use string literals inside the list of facts.